packages/@react-spectrum/dialog/stories/DialogTrigger.stories.tsx

```diff
@@ -331,6 +331,26 @@ storiesOf('DialogTrigger', module)
         </Flex>
       </Flex>
     )
+  )
+  .add(
+    '2 popovers',
```

Comment:
I am able to open both popovers at the same time. Is that intended?

Reply:
definitely not intentional and i had it fixed at one point
i'll check out what happened and try to get that to work correctly again